rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/10_basic.yml

```diff
@@ -54,3 +54,51 @@
 
   - match: { status: 400 }
   - match: { error.root_cause.0.type: "illegal_argument_exception" }
+
```

Comment:
I think we can tweak `ESRestTestCase#wipeCluster()` to now also delete all data streams.
(in the same way component clusters are removed, which takes into account that the feature may not be enabled.